src/screen/home.js

```diff
@@ -72,9 +75,16 @@ function App() {
     document.body.classList.add("app");
     document.title = "Weather";
     window.navigator.geolocation.getCurrentPosition((position) => {
-      fetchWeather(`https://api.openweathermap.org/data/2.5/weather?lat=${position.coords.latitude}&lon=${position.coords.longitude}&appid=${process.env.REACT_APP_APIKEY}`)
-      
+      fetchWeather(`https://api.openweathermap.org/data/2.5/weather?lat=${position.coords.latitude}&lon=${position.coords.longitude}&appid=${process.env.REACT_APP_APIKEY}`)  
```

Comment:
Is this function calling itself? Maybe two other issues (#20 and #27) have taken care of this so you might want to check that.

Reply:
@joshi-kaushal are you referring to the fetchWeather()? Because it does not seem to call itself and its the same in other issues too because we did not write this line of code particularly maybe changed the format of the line while adding stuff to useEffect

Reply:
That's why I thought you are calling the same function inside itself.

![image](https://user-images.githubusercontent.com/53049546/181734526-9b0e9056-4b76-41e6-bf51-aa1a7a6d9565.png)

Reply:
I don't believe its inside the body of the function. nvm, it is not being called by itself